sdk/core/Azure.Core/tests/HttpPipelineMessageTest.cs

```diff
@@ -100,5 +102,33 @@ public void ContentPropertyThrowsResponseIsExtracted()
             Assert.AreSame(memoryStream, stream);
             Assert.Throws<InvalidOperationException>(() => { var x = response.Content; });
         }
+
+        [Test]
+        public async Task UnbufferedStreamAccessibleAfterMessageDisposed()
```

Comment:
Bringing this in from https://gist.github.com/AlexanderSher/df22ed21acaa938a4371c188f0699b11